projects/storefrontlib/src/lib/cart/facade/cart.service.ts

```diff
@@ -17,21 +17,21 @@ import { ANONYMOUS_USERID, CartDataService } from './cart-data.service';
 export class CartService {
   private callback: Function;
 
```

Comment:
Please move methods under the constructor

Reply:
Rename `initCart` to `init`. Make it `protected`

Reply:
Rename `loadCartDetails` -> `load`. Add `void`.

Reply:
Rename `addCartEntry` to `addEntry`. Add `void`.

Reply:
Rename `refreshCart` to `refresh`. Make it `protected`.

Reply:
Rename `loadOrMergeCart` to `loadOrMerge`

Reply:
In short:
* make all `private` methods `protected`
* make sure there's a return type
* avoid using `cart` noun in the method name.